src/transactions/feed/TransactionFeedV2.tsx

```diff
@@ -63,6 +190,14 @@ export default function TransactionFeedV2() {
       selectFromResult: (result) => {
         return {
           ...result,
+          /**
+           * Under the hood, selectFromResult is passed to the "createSelector" function so we need
+           * to memoize any processed data in order to keep the reference. We could move the whole
+           * implementation outside of the component into a pure function but then we'll lose the type inference.
+           *
+           * Considering this function changes the type definition of useTransactionFeedV2Query result
+           * object - it is easier to use "useMemo" to keep the type-safety.
+           */
```

Comment:
Sorry I'm not sure I fully understand the details here.
This `useMemo` so far feels like a hack to me.
I guess we should chat directly so I can understand this.